PC/bdist_wininst/install.c

```diff
@@ -631,7 +631,7 @@ static PyObject *PyMessageBox(PyObject *self, PyObject *args)
     return g_Py_BuildValue("i", rc);
 }
 
-static PyObject *GetRootHKey(PyObject *self)
+static PyObject *GetRootHKey(PyObject *self, PyObject *Py_UNUSED(ignored))
```

Comment:
Casting to `PyCFunction` below no longer needed. And this is the only use of `PyCFunction`, thus its declaration can be removed.

Reply:
Ah I missed this one, I'll fix it.